config/webpack.config.js

```diff
@@ -80,7 +83,7 @@ var configGenerator = (options) => {
     },
     plugins: [
       new webpack.DefinePlugin({
-          __BUILDTYPE__: options.buildtype
+          __BUILDTYPE__: JSON.stringify(options.buildtype)
```

Comment:
Is `process.env.NODE_ENV` set to `"production"` in any of the Webpack configs? It seems like this is building React (and other libs) in dev mode all the time (and there's an error to that effect on the prod vets.gov site).

(I'm new, so apologies if this is taken care of elsewhere or already been talked about)